tools/gen_addon_readme.py

```diff
@@ -14,7 +14,6 @@
 
 from .gitutils import commit_if_needed
 from .manifest import read_manifest, find_addons, NoManifestFound
-from .runbot_ids import get_runbot_ids
```

Comment:
can't we trash related files?

Reply:
I'm not aware of other processes that could use them. I just limited to gen_addon_readme changes.